code/game/objects/effects/decals/posters/bs12.dm

```diff
@@ -293,3 +293,40 @@
 	icon_state="bsposter60"
 	name = "John Fleet poster"
 	desc = "A poster produced by the SCGF. It depicts Petty Officer John Fleet, a character used to boost troop morale, standing in front of the SCG flag proudly. Nobody knows if he really existed."
+
+///TeguStation posters
+
+/// Moth poster sprites are created by Azlanon and AyyRobotics
```

Comment:
@Cupax3 please clarify if modularization comments are needed

Reply:
Nice to have, not required